elliottlib/cli/find_bugs_sweep_cli.py

```diff
@@ -279,7 +279,9 @@ def categorize_bugs_by_type(bugs: List[Bug], advisory_id_map: Dict[str, int], ma
         attached_builds = errata.get_advisory_nvrs(advisory_id_map[kind])
         packages = list(attached_builds.keys())
         if kind == 'image':
-            packages.extend(constants.SPECIAL_CVE_COMPONENTS)
+            # golang builder is a special tracker component
+            # which applies to all our golang images
+            packages.append(constants.GOLANG_BUILDER_CVE_COMPONENT)
```

Comment:
Keeping this simple - don't think we're going to have any special cve component except golang builder